File: Python/stealth_playwright.py
from playwright.sync_api import sync_playwright
from playwright_stealth import stealth_sync

with sync_playwright() as p:
    browser = p.chromium.launch(headless=True)
    context = browser.new_context()
    page = context.new_page()

    stealth_sync(page)  

    page.goto("https://example.com", timeout=60000)
    print(page.title())

    browser.close()

File: NodeJS/stealth_playwright.js
const { chromium } = require('playwright-extra');
const stealth = require('playwright-extra-plugin-stealth')();

chromium.use(stealth);

(async () => {
  const browser = await chromium.launch({ headless: true });
  const context = await browser.newContext();
  const page = await context.newPage();

  await page.goto('https://example.com', { timeout: 60000 });
  const title = await page.title();
  console.log(title);

  await browser.close();
})();